modules/fb_instant_articles_api_rules/fb_instant_articles_api_rules.module

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @file
+ * Provide Drupal rules to manage facebook instant article using API.
+ */
+
+/**
+ * Implements hook_action_info().
+ */
+function fb_instant_articles_api_rules_action_info() {
+  return array(
+    'facebook_instant_articles_api_import_action' => array(
+      'type' => 'node',
+      'label' => t('Add facebook instant article'),
+      'configurable' => FALSE,
+      'triggers' => array(
+        'node_presave', 'node_insert', 'node_update',
+      ),
+    ),
+    'facebook_instant_articles_api_remove_action' => array(
+      'type' => 'node',
+      'label' => t('Delete facebook instant article'),
+      'configurable' => FALSE,
+      'triggers' => array(
+        'node_delete',
+      ),
+    ),
+  );
+}
+
+
+/**
+ * Implementation of a Facebook instant article node import action.
+ *
+ */
+function facebook_instant_articles_api_import_action($node, $context = array()) {
+  fb_instant_articles_api_rules_import_article($node);
+}
+
+
+
+/**
+ * Implementation of a Facebook instant article node delete action.
+ *
+ */
+function facebook_instant_articles_api_remove_action($node, $context = array()) {
+  fb_instant_articles_api_rules_remove_article($node);
+}
+
+/**
+ * Implementation of a Facebook instant article import.
+ *
+ *@param object $node
+ */
+function fb_instant_articles_api_rules_import_article($node) {
+  if (isset($node->nid)) {
+    $layout_settings = fb_instant_articles_display_get_node_layout_settings($node->type);
+    $wrapper = InstantArticleNodeWrapper::create($node, $layout_settings);
+    $node->fb_instant_articles_display_wrapper = $wrapper;
+    $node_view = node_view($node, $view_mode = 'fb_instant_article');
+    $import_article = drupal_render($node_view);
+    try {
+      Drupal\FBInstantArticles\DrupalClient::importArticle($import_article, $node->status);
+    }
+    catch(Facebook\Exceptions\FacebookAuthorizationException $e) {
+      watchdog_exception('facebook_instant_article', $e);
```

Comment:
Any logging via the API calls should be added to the API module. See [fb_instant_articles_display_init()](https://github.com/BurdaMagazinOrg/module-fb_instant_articles/blob/7.x-1.x/modules/fb_instant_articles_display/fb_instant_articles_display.module#L8) for the current technique. This is a good thought though, because:

1. We should open a new issue on d.o to add Drupal logging relevant to the API module.
2. We should open another issue on d.o to move the `LoggerAppenderDrupal` class file, and perhaps the `fb_instant_articles_display_enable_transformer_logging` variable to a setting across the board for all of the instant articles modules (and then each submodule can implement it's own hook to register the relevant SDK loggers appropriate to each).